doc/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender/index.md

```diff
@@ -0,0 +1,720 @@
+---
+layout: page
+title: Cassandra log appender
+permalink: /:path/
+nav: /:path/Programming-guide/Key-platform-features/Data-collection/Cassandra-log-appender
+sort_idx: 10
+---
+
+{% assign root_url = page.url | split: '/'%}
+{% capture root_url  %} /{{root_url[1]}}/{{root_url[2]}}/{% endcapture %}
+
+* [Creating Cassandra log appender with Admin UI](#creating-cassandra-log-appender-with-admin-ui)
+* [Creating Cassandra log appender with REST API](#creating-cassandra-log-appender-with-rest-api)
+  * [Configuration](#configuration)
+  * [Administration](#administration)
+* [Playing with Cassandra log appender](#playing-with-cassandra-log-appender)
+
+The Cassandra log appender is responsible for transferring logs from the Operations server to the Cassandra database.
+
+### Creating Cassandra log appender with Admin UI
+
+The easiest way to create a Cassandra log appender for your application is by using Admin UI.
+
+To create a log appender do the following:
+
+1. In the <b>Log appenders</b> window, click <b>Add log appender</b>.
+2. Enter the log appender name and description, select the minimum and maximum supported schema version, select necessary log metadata fields.
+3. Set the log appender type to <i>Cassandra</i>.
+4. Fill in the Cassandra log appender configuration form.
+5. Click <b>Add</b>.
+
+<img src="attach/create-cassandra-log-appender-admin-ui.png">
+
+### Creating Cassandra log appender with REST API
+
+It is also possible to create a Cassandra log appender instance by using [REST API]({{root_url}}Programming-guide/Server-REST-APIs). The following example 
+illustrates how to provision the Cassandra log appender via REST API.
+
+### Configuration
+
+The Cassandra log appender configuration should match the following Avro schema:
+
+```json
+{
```

Comment:
I think this schema and all other schemes should be a link to github file (repository). No need to copy-paste schemes here.